mocks/router/middlewares/JSONHandler.go

```diff
@@ -20,6 +20,10 @@ func (_m *JSONHandler) Execute(w http.ResponseWriter, r *http.Request) (interfac
 	ret := _m.Called(w, r)
 
 	var r0 interface{}
+	var r1 httperrors.HTTPError
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.